tests/feature_extraction/test_feature_calculations.py

```diff
@@ -568,22 +575,52 @@ def test_friedrich_coefficients(self):
         six.assertCountEqual(self, list(res.index), expected_index)
         self.assertTrue(np.sum(np.isnan(res)), 3)
 
-        
     def test_max_langevin_fixed_point(self):
         """
         Estimating the intrinsic velocity of a dissipative soliton
         """
         default_params = {"m": 3, "r": 30}
-        
+
         # active Brownian motion
         ds = velocity(tau=3.8, delta_t=0.05, R=3e-4, seed=0)
         v = ds.simulate(1000000, v0=np.zeros(1))
-        v0 = max_langevin_fixed_point(v[:,0], **default_params)
-        self.assertTrue(abs(ds.deterministic-v0)<0.0001)
+        v0 = max_langevin_fixed_point(v[:, 0], **default_params)
+        self.assertTrue(abs(ds.deterministic - v0) < 0.0001)
 
         # Brownian motion
-        ds = velocity(tau=2.0/0.3-3.8, delta_t=0.05, R=3e-4, seed=0)
+        ds = velocity(tau=2.0 / 0.3 - 3.8, delta_t=0.05, R=3e-4, seed=0)
         v = ds.simulate(1000000, v0=np.zeros(1))
-        v0 = max_langevin_fixed_point(v[:,0], **default_params)
-        self.assertTrue(v0<0.001)
+        v0 = max_langevin_fixed_point(v[:, 0], **default_params)
+        self.assertTrue(v0 < 0.001)
```

Comment:
Yes, I know. You wanted to adjust the overall code style.... But, as I always say, please use another PR for this :-P ;-)